analytics/pyproject.toml

```diff
@@ -8,7 +8,6 @@ version = "0.1.0"
 
 [tool.poetry.scripts]
 analytics = "analytics.cli:app"
-db-migrate = "src.analytics.cli:migrate_database"
```

Comment:
Removed this since the same functionality is available with `etl db_migrate` and I was having trouble exposing multiple entry points in the docker image.

Reply:
Unfortunately you will in fact need to put this one back. The problem you say likely stems from using ENTRYPOINT and the end of your docker file rather than CMD. So I would change it to CMD, and update your manual tests accordingly

Reply:
You'll notice, for example, that the API uses CMD

(Ignore the fact that's its using poetry heh)

https://github.com/HHS/simpler-grants-gov/blob/main/api/Dockerfile#L127

Reply:
Added this back and updates the Dockerfile to _not_ use a default entry point so that it can support both:

`docker run <image> db-migrate` and `docker run <image> analytics etl <sub-command>`